winlogtimeline/ui/import_wizard.py

```diff
@@ -0,0 +1,117 @@
+        # Container frame
+        self.container = Frame(self)
+
+        # Workspace block
+        self.sv_file = StringVar()
+        self.label_file = Label(self.container, text='File:')
+        self.entry_file = Entry(self.container, textvariable=self.sv_file)
+        self.button_file = Button(self.container, text='...', width=1, command=self.callback_path_prompt)
+
+        # Alias block
+        self.sv_alias = StringVar()
+        self.label_alias = Label(self.container, text='Alias:')
+        self.entry_alias = Entry(self.container, textvariable=self.sv_alias)
+
+        # Action block
+        self.button_import = Button(self.container, text='Import', underline=0, command=self.callback_import)
+        self.button_cancel = Button(self.container, text='Cancel', underline=0, command=self.callback_cancel)
+        self.bind('<Alt-i>', self.callback_import)
+        self.bind('<Alt-c>', self.callback_cancel)
+
+        # Default values
+        self.sv_file.set(os.path.join(util.data.get_appdir(), ''))
+        self.sv_alias.set('Alias')
+
+    def _place_widgets(self):
+        """
+        Lays out the elements in this window.
+        :return:
+        """
+        padding = 3
+
+        # Workspace block
+        self.label_file.grid(row=0, column=0, padx=padding, sticky='SW')
+        self.entry_file.grid(row=1, column=0, columnspan=4, padx=padding, pady=padding, sticky='NESW')
+        self.button_file.grid(row=1, column=5, padx=padding, pady=padding, sticky='EW')
+
+        # Title block
+        self.label_alias.grid(row=2, column=0, padx=padding, sticky='SW')
+        self.entry_alias.grid(row=3, column=0, padx=padding, pady=padding, sticky='NESW')
+
+        # Action block
+        self.button_cancel.grid(row=6, column=2, padx=padding, pady=padding, sticky='EW')
+        self.button_import.grid(row=6, column=3, padx=padding, pady=padding, sticky='EW')
+
+        # Specify which portion to auto-expand
+        self.columnconfigure(0, weight=4)
+
+        # Place the container frame
+        self.container.pack(fill=BOTH)
+
+    def callback_path_prompt(self):
+        """
+        Callback used to kick off a directory selection prompt.
+        :return:
+        """
+        file = filedialog.askopenfilename(title='Choose an Event Log File',
+                                          filetypes=(('Windows Event Log File', '*.evtx'),))
+        if len(file) > 0:
+            self.sv_file.set(os.path.abspath(file))
+
+    def callback_import(self, event=None):
+        """
+        Callback used when the import button is pressed. Ensures that the path is valid and performs error handling.
+        :return:
+        """
+        record_file = os.path.abspath(self.sv_file.get())
+        if not os.path.isfile(record_file):  # Does the file exist.
+            messagebox.showerror('Error', "The file doesn't exists:\n{}".format(record_file))
+            return
+
+        # call import function here -> calls from the ui.py
```

Comment:
Do some validation on the alias here. Make sure it's not empty and that it doesn't already exist in the database.